### PR TITLE
Create list recommendations feature page

### PR DESCRIPTION
**Pull request by brianshao-prog**
### Pre-Requisites

- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [ ] Yes, I included and/or modified tests to cover relevant code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

### Description of Changes

Within web\src\pages\recommendations, created a recommendations page to display dummy links from links.json, made recommendation component module css to make unordered lists unbulleted

### Related Issues

Related to issue #8

### CODE REVIEW

File: packages/web/src/pages/recommendations/index.tsx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+  HStack,
+  Image,
+  Link,
+  Stack,
+  Text,
+  useColorModeValue,
+  VStack,
+} from '@chakra-ui/react';
+import { CircularProgress, CircularProgressLabel } from '@chakra-ui/react';
+
```
Comment (Ulfingrad) [inline packages/web/src/pages/recommendations/index.tsx:18]:
```suggestion
```

File: packages/web/src/pages/recommendations/index.tsx [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+import { MarketingLayout } from '../../components/Layout';
+
```
Comment (Ulfingrad) [inline packages/web/src/pages/recommendations/index.tsx:20]:
```suggestion
```

**Review by Ulfingrad — CHANGES_REQUESTED** [review]
Great start! Dre and I will give it another passthrough once these changes are made

File: packages/web/src/pages/recommendations/index.tsx [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+        </Center>
+
+        <HStack spacing={8}>
+          <Button as="a" size="lg" colorScheme="blue" href="https://nextjs.org/" target="_blank">
+            Next.js
+          </Button>
+          <Button as="a" size="lg" colorScheme="gray" href="https://chakra-ui.com/" target="_blank">
+            Chakra UI
+          </Button>
+        </HStack>
```
[the remark this answers is not in the document]
Comment (brianshao-prog) [reply packages/web/src/pages/recommendations/index.tsx:290]: Will keep it in for now

File: packages/web/src/pages/recommendations/index.tsx [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+      <VStack alignItems="center" paddingY={32} spacing={8}>
+        <Heading size="4xl" textAlign="center" lineHeight="1.4">
+          The Web App Template of the recommendations list
+        </Heading>
+        <Text textAlign="center" maxWidth="60ch" fontSize="2xl" lineHeight="2">
+          You&apos;ve got a production ready React.js frontend (with Dark Mode) and Express.js
+          backend. It is written in Typescript and has 100% test coverage.
+          <br />
+          Now go build something cool!
+        </Text>
```
[the remark this answers is not in the document]
Comment (brianshao-prog) [reply packages/web/src/pages/recommendations/index.tsx:66]: Will keep it in for now, just need to merge in the code in order to progress with rest of the team

**Review by gageayk — APPROVED** [review]
Good to go for now, testing will be fixed after workshop.

**Pull request merged**